tests/h/activity/bucketing_test.py

```diff
@@ -13,20 +15,10 @@
 FIFTH_NOVEMBER_1969 = datetime.datetime(year=1969, month=11, day=5)
 
 
-class timeframe_with:
-    def __init__(self, label, document_buckets):
-        self.label = label
-        self.document_buckets = document_buckets
-
-    def __eq__(self, timeframe):
-        return (
-            self.label == timeframe.label
-            and self.document_buckets == timeframe.document_buckets
-        )
-
-    def __repr__(self):
-        # pragma: no cover
-        return f'{self.__class__} "{self.label}" with {len(self.document_buckets)} document buckets'
+def timeframe_with(label, document_buckets):
+    return Any.instance_of(Timeframe).with_attrs(
+        {"label": label, "document_buckets": document_buckets}
+    )
```

Comment:
I wonder if we want to go on a general hunt and remove all the matcher stuff from `h`. I'm pretty sure it currently falls into two categories:

 * Things `h-matchers` can do
 * Things `h-matchers` should be upgraded to do if it can't